workflows/ascc_organellar.nf

```diff
@@ -284,7 +284,7 @@ workflow ASCC_ORGANELLAR {
         //
         .map{ meta, file ->
             def total_length = 0
-            new File(fasta_file).eachLine { line ->
+            new File(file).eachLine { line ->
```

Comment:
Can this be moved to a) a native `exec:` process (allows caching), and b) use the `splitFasta()` function? https://www.nextflow.io/docs/latest/reference/operator.html#splitfasta

Something like:

```
file
    .splitFasta(by: 1, record: [seqString: true])
    .collect { it.seqString.size() }
    .sum() 
```

Reply:
(maybe that loads the whole FASTA into memory, though...)

Reply:
Sorry, your comment has only just loaded for me.

I think it might, I *think* eachLine is better from a memory perspective. I just needed to remove the `new File`, it just doesn't support files pulled from internet.

It could just be its own process, I've just always figured that it's such a simple job it shouldn't need it.

```
file.eachLine { line ->
                if (line && !line.startsWith('>')) {
                    total_length += line.length()
                }
            }
```

Reply:
In any case it's just a thought, not necessary for this PR!